clang/lib/Sema/SemaExprCXX.cpp

```diff
@@ -5012,6 +5012,20 @@ Sema::PerformImplicitConversion(Expr *From, QualType ToType,
   return From;
 }
 
+/// Checks that type T is not a VLA.
+///
+/// @returns @c true if @p T is VLA and a diagnostic was emitted,
+/// @c false otherwise.
+static bool DiagnoseVLAInCXXTypeTrait(Sema &S, const TypeSourceInfo *T,
```

Comment:
nit: do we need the `CXX` in the function name here?

Reply:
I don't have a strong opinion on this. Just followed what Corentin suggested.

Reply:
Ah, I see; the main reason I commented on this is because I thought it was a bit odd that none of the `EvaluateXYTypeTrait` functions have `CXX` in them, but this is one does (I know we don’t really have fixed ‘rules’ as to when to include `CXX` in the name, but it just wanted to suggest maybe being consistent about it at least). I don’t feel strongly about this either.

Reply:
One reason might be the fact that current use case for this function is C++ traits specifically, since VLA is clearly an extension for them. I'll keep the `CXX` then.